dts/arm/st/l5/stm32l5.dtsi

```diff
@@ -584,12 +584,25 @@
 			clocks = <&rcc STM32_CLOCK_BUS_AHB2 0x00002000>;
 			interrupts = <37 0>;
 			status = "disabled";
+			vref-mv = <3000>;
 			label = "ADC_1";
 			#io-channel-cells = <1>;
 			has-temp-channel;
 			has-vref-channel;
 		};
 
+		die_temp: dietemp {
+			compatible = "st,stm32-temp-cal";
+			ts-cal1-addr = <0x0BFA05A8>;
+			ts-cal2-addr = <0x0BFA05CA>;
+			ts-cal1-temp = <30>;
+			ts-cal2-temp = <110>;
```

Comment:
From datasheet, DS12737 Rev 6, Cal2 temp is 130°c not 110:
![image](https://user-images.githubusercontent.com/50730894/178760314-0879c3d5-5acd-49b8-95ff-0d522e69cf81.png)